src/main/js/controls/PairedResult/helpers/helpers.js

```diff
@@ -431,7 +432,9 @@ const processRegions = (graphContext, config, canvasSVG, { key }) => {
             config.shownTargets.indexOf(key) > -1
         );
     } else {
-        hideAllRegions(canvasSVG);
+        config.isHideAllRegion === false && config.shownTargets.length > 0
```

Comment:
the if condition is checking only for graphcontent.length === 1 so for combination(Line,PairedResult) it will always go in the else and adding the config.shownTargets.length > 0 is because if all legends are clicked ( config.shownTargets.length = 0)and single legend is clicked again (config.shownTargets.length = 1) then also we have to show the region if it is same ( only combination)

Reply:
I was more expecting a unit test for this. But this is also connected to the bug
https://github.com/cerner/carbon-graphs/issues/3
Ill re-review this once you post your changes.